src/main/java/nextstep/subway/domain/StationRepository.java

```diff
@@ -1,10 +1,13 @@
 package nextstep.subway.domain;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.query.Param;
 
 import java.util.List;
 
 public interface StationRepository extends JpaRepository<Station, Long> {
     @Override
     List<Station> findAll();
+    List<Station> findAllByLineIdIsNotNull();
+    List<Station> findAllByLineId(@Param(value = "lineId")Long lineId);
```

Comment:
```suggestion
    List<Station> findAllByLineId(Long lineId);
```

@Param은 생략해주셔도 될 것 같네요!